pkg/operator/awspodidentity/controller.go

```diff
@@ -109,7 +109,8 @@ func Add(mgr manager.Manager, kubeconfig string) error {
 	logger := log.WithFields(log.Fields{"controller": controllerName})
 	imagePullSpec := os.Getenv("AWS_POD_IDENTITY_WEBHOOK_IMAGE")
 	if len(imagePullSpec) == 0 {
-		return errors.New("AWS_POD_IDENTITY_WEBHOOK_IMAGE is not set")
+		logger.Warn("AWS_POD_IDENTITY_WEBHOOK_IMAGE is not set, skipping controller")
```

Comment:
Do we need doc update here saying we need to set 'AWS_POD_IDENTITY_WEBHOOK_IMAGE' environment variable while running from source?

Reply:
I'm not sure just yet, I would like this to come up smoothly without having to do anything on 4.4, we probably need a separate PR to make that happen, there's an additional problem area I had to hack around to get it working.

Reply:
i'll open the PR for this targeted change that also fixes https://bugzilla.redhat.com/show_bug.cgi?id=1846200